src/LocalizedStrftimeFormatter.php

```diff
@@ -152,7 +152,7 @@ private function strftime(string $format, DateTimeImmutable $timestamp, string $
 
 			$replace = $translationTable[$pattern];
 
-			if (is_callable($replace)) {
+			if (!is_string($replace) && is_callable($replace)) {
```

Comment:
Can we invert this check to
```
			if (is_string($replace)) {
				$result = $timestamp->format($replace);
			}
			else {
				$result = $replace($timestamp, $pattern);
			}
```

I think this is more readable and phpstan does the type check that the reset of the stuff in `$translationTable` is callable

Also can you fix the phpstan issue by adding `(string)` on line `164` and `168`

Reply:
@sunkan I will invert the check. It makes much more sense that way.

The phpstan issue is actually fixed by updating phpstan, but it creates a new problem. A weird one that I am not sure how is best resolved.